src/templates/drizzle/logo.hbs

```diff
@@ -1 +1,7 @@
-Drizzle
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60" width="30" fill="currentColor" class="drizzle-u-marginRight">
+  <title>Drizzle Icon</title>
```

Comment:
This is probably unnecessary for screen readers since it accompanies some Drizzle text. I would consider removing it and adding `role="presentation"` to the `<svg>` element instead.